### PR TITLE
pl, cs & ru locales update & utf8 fix

### PR DESCRIPTION
**Pull request by JustOff**

### CODE REVIEW

**Comment by janekptacijarabaci** [comment]
Ad #147 
Thank you. Already it looks good :-)

File: en-US/chrome/en-US/locale/en-US/global/dom/dom.properties [hunk 1]
```diff
@@ -137,11 +137,11 @@ GetPreventDefaultWarning=Use of getPreventDefault() is deprecated.  Use defaultP
 # LOCALIZATION NOTE: Do not translate "getUserData", "setUserData", "WeakMap", or "element.dataset".
 GetSetUserDataWarning=Use of getUserData() or setUserData() is deprecated.  Use WeakMap or element.dataset instead.
 # LOCALIZATION NOTE: Do not translate "mozGetAsFile" or "toBlob"
-MozGetAsFileWarning=The non-standard mozGetAsFile method is deprecated and will soon be removed.  Use the standard toBlob method instead.
+MozGetAsFileWarning=The non-standard mozGetAsFile method is deprecated and will soon be removed. Use the standard toBlob method instead.
 # LOCALIZATION NOTE: Do not translate "captureEvents()" or "addEventListener()"
 UseOfCaptureEventsWarning=Use of captureEvents() is deprecated. To upgrade your code, use the DOM 2 addEventListener() method. For more help http://developer.mozilla.org/en/docs/DOM:element.addEventListener
```
Comment (janekptacijarabaci) [inline en-US/chrome/en-US/locale/en-US/global/dom/dom.properties]: The same is valid here...

From:
http://developer.mozilla.org/en/docs/DOM:element.addEventListener
To:
https://developer.mozilla.org/docs/Web/API/EventTarget/addEventListener

Comment (janekptacijarabaci) [inline en-US/chrome/en-US/locale/en-US/global/dom/dom.properties]: The same is valid here...

From:
http://developer.mozilla.org/en/docs/DOM:element.addEventListener
To:
https://developer.mozilla.org/docs/Web/API/EventTarget/addEventListener

**Comment by wolfbeast** [comment]
Thanks!

**Pull request merged**